src/backend/distributed/commands/trigger.c

```diff
@@ -0,0 +1,187 @@
+/*-------------------------------------------------------------------------
+ * trigger.c
+ *
+ * This file contains functions to create and process trigger objects on
+ * citus tables.
+ *
+ * Copyright (c) Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+#include "postgres.h"
+#include "distributed/pg_version_constants.h"
+
+#include "access/genam.h"
+#if PG_VERSION_NUM >= PG_VERSION_12
+#include "access/table.h"
+#else
+#include "access/heapam.h"
+#include "access/htup_details.h"
+#endif
+#include "catalog/indexing.h"
+#include "catalog/namespace.h"
+#include "catalog/pg_trigger.h"
+#include "distributed/citus_ruleutils.h"
+#include "distributed/commands.h"
+#include "distributed/listutils.h"
+#include "distributed/metadata_cache.h"
+#include "utils/fmgroids.h"
+#include "utils/lsyscache.h"
+
+/*
+ * GetExplicitTriggerCommands returns the list of DDL commands to (re)create
```

Comment:
minor: As far as I can see, we always `create` triggers, as there is no `create or replace` command. So, remove  `(re)` part